janusgraph-cassandra/src/main/java/org/janusgraph/diskstorage/cassandra/astyanax/AstyanaxStoreManager.java

```diff
@@ -285,6 +289,7 @@
 
     public AstyanaxStoreManager(Configuration config) throws BackendException {
         super(config);
+        log.warn("Astyanax is deprecated and will be removed with JanusGraph 0.5.0. Please consider switching to the CQL backend.");
```

Comment:
Switching is non-optional (since Astyanax will be removed in 0.5.0), so instead of a very gentle "please consider switching to ...", it should be a more explicit statement of "please switch to ...".